tests/e2e/server/index.js

```diff
@@ -0,0 +1,40 @@
+import { Server, Response } from 'miragejs';
+import { accessToken, initial, second, userInfo } from './data';
+
+new Server({
+  routes() {
+    this.urlPrefix = 'https://forgerock-sdk-samples.com:3000';
```

Comment:
Does this need to be the same domain for CORS or just simplicity of only faking one domain?  If not, consider using a separate domain for the fake AM server so it's immediately clear what this is.

Reply:
It was for solely just having to fake just one domain, but they do have different ports, so CORS is still at play here. I wanted to use different domains, but then we'd have to add yet another alias to our `etc/hosts` file. Do you think it's worth the effort, or is this fine?

Reply:
I think it's worth adding the extra alias to `hosts`, which we can simplify in the readme with something like:

```
echo '127.0.0.1 forgerock-sdk-samples.com openam-sdk-samples.com' | sudo tee -a /etc/hosts
```

Reply:
Okay, I can do that.